src/rebuild/scan.c

```diff
@@ -581,125 +690,53 @@
 	crt_group_rank(rpt->rt_pool->sp_group, &myrank);
 	md.omd_ver = rpt->rt_rebuild_ver;
 	md.omd_fdom_lvl = arg->co_props.dcp_redun_lvl;
+	tgts = tgt_array;
+	shards = shard_array;
+	switch(rpt->rt_rebuild_op) {
```

Comment:
(style)  switch and case should be at the same indent
(style)  space required before the open parenthesis '('